tests/integration/api/__init__.py

```diff
@@ -0,0 +1,2 @@
+# Copyright (C) 2023 Intel Corporation
```

Comment:
```suggestion
# Copyright (C) 2024 Intel Corporation
```

Need to change the year of the copyright. (also exists at other codes)

Reply:
Thanks! Done - https://github.com/openvinotoolkit/training_extensions/commit/90ef60e2a4a5f817c332b762cb8ec20f0e5201b7